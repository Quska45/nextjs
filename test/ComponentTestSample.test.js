/**
 * @jest-environment jsdom
 */

import ComponentTestSample from "./ComponentTestSample";
import { render } from "@testing-library/react"

it("matches snapshot", () => {
    const utils = render(<ComponentTestSample name="호박너구리" mbti="ESFJ"></ComponentTestSample>)
    expect(utils.container).toMatchSnapshot()
})