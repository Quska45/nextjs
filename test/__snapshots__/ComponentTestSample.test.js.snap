// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div>
    <h2>
      호박너구리
    </h2>
    <p>
      ESFJ
    </p>
  </div>
</div>
`;
